rollup/internal/utils/utils.go

```diff
@@ -91,7 +93,12 @@ func CalculateChunkMetrics(chunk *encoding.Chunk, codecVersion encoding.CodecVer
 		metrics.L1CommitUncompressedBatchBytesSize, metrics.L1CommitBlobSize, err = codecv2.EstimateChunkL1CommitBatchSizeAndBlobSize(chunk)
 		metrics.EstimateBlobSizeTime = time.Since(start)
 		if err != nil {
-			return nil, fmt.Errorf("failed to estimate codecv2 chunk L1 commit blob size: %w", err)
+			if errors.Is(err, &encoding.CompressedDataCompatibilityError{}) {
+				log.Error("Compressed data compatibility check failed", "error", err)
```

Comment:
Why log an error here?

First of all this is already logger in `da-codec`.

Second, we don't know here if this is an error. We'll inspect and log accordingly in higher layers (chunk-proposer, etc.)

Reply:
remove the `log.Error` and add `log.Warn` logs both in `chunk-proposer` and `batch-proposer` to print more info to track the chunk & batch proposed due to violation.